plugin.video.putio/resources/__init__.py

```diff
@@ -0,0 +1,16 @@
+# put.io kodi addon
```

Comment:
should not be needed anymore with python3 please drop it

Reply:
f1a18eac